.env.template

```diff
@@ -5,4 +5,7 @@
 # https://developers.google.com/sheets/api/quickstart/js
 
 API_KEY= YOUR_KEY_HERE
-CLIENT_ID= YOUR_CLIENT_HERE
\ No newline at end of file
+CLIENT_ID= YOUR_CLIENT_HERE
+
+# Used to enable logging for developers
+DEVELOPMENT= true
```

Comment:
Can we omit this environment variable?

How about development is on when `production` is false? `rollup.config.js` already has a variable that keeps  track of production.

Reply:
Good catch! I've made have now made this change